src/legacy/core_plugins/kibana/public/discover/components/doc/use_es_doc_search.test.tsx

```diff
@@ -43,7 +43,7 @@ describe('Test of <Doc /> helper / hook', () => {
     `);
   });
 
-  test('useEsDocSearch', () => {
+  test('useEsDocSearch', async () => {
```

Comment:
This is just a small cleanup getting rid of the `async state update outside of act(...)` warnings.
The latest react update made it possible to wait for the async updates by using asynchronous `act`. In the case of this test the assertions assumed no async functions would be called, so the mocks had to be slightly adjusted.